src/modules/back-to-top/back-to-top.component.html

```diff
@@ -0,0 +1,3 @@
+<div #backToTop class="back-to-top" (click)="goToTop()" [ngStyle]="{'background-color': backgroundColor, 'color': color}">
```

Comment:
It could still be in a spa, but maybe they want to use the blackbaud green (or if we are being open source minded, their company's color), I think we should give them the option right?